pkg/controller/deployment/sync.go

```diff
@@ -199,7 +199,7 @@ func (dc *DeploymentController) addHashKeyToRSAndPods(rs *extensions.ReplicaSet)
 	// 2. Update all pods managed by the rs to have the new hash label, so they will be correctly adopted.
 	selector, err := unversioned.LabelSelectorAsSelector(updatedRS.Spec.Selector)
 	if err != nil {
-		return nil, fmt.Errorf("error in converting selector to label selector for replica set %s: %s", updatedRS.Name, err)
+		return nil, fmt.Errorf("error in converting selector to label selector for ReplicaSet %s: %s", updatedRS.Name, err)
```

Comment:
I was fixing more of ReplicaSet as being a single word not two words. Label Selector is two words. replica set is not two words is my reasoning. @kubernetes/deployment does every one agree ?